index.html

```diff
@@ -15,28 +15,38 @@
     <header class="header">
         <a href="#" class="logo">Alpha Portifolio</a>
         <nav><i class="fa fa-bars"></i></nav>
+        <ul class="header-links">
+            <li><a href="#">Portifolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
     </header>
     <main>
 
         <!-- THIS IS A SECTION FOR MOBILE PORTIFOLIO SKELETON -->
         <section class="hdln-section">
-            <h1 class="name-intro">
-                I’m Alpha <br>
-                Glad to see <br> you!
-            </h1>
-            <P class="personal-info">
-                I’m a software developer! I can help you build a product , feature or
-                website Look through some of my work and experience! If you like
-                what you see and have a project you need coded, don’t hestiate to contact me.
-            </P>
-            <div class="connect"><a href="#">LET'S CONNECT</a> </div>
-            <ul class="social">
-                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-                <li><a href="#"><img src="./images/Vector.svg" alt="M icon"></a></li>
-                <li><a href="#"><img src="./images/VectorGit.svg" alt="github icon"></a></li>
-                <li><img src="./images/VectorImoji.svg" alt="peace imoji"></li>
-            </ul>
+            <div class="hdln-img-container">
+                <img src="./images/header-shapes mobile.png" alt="">
+            </div>
+            <div class="banner-text-container">
+                <h1 class="name-intro">
+                    I’m Alpha <br>
+                    Glad to see <br> you!
```

Comment:
- [ ] (OPTIONAL) Kindly apply the correct font color and font family to your headings 👍 
- Your design:
![Annotation 2022-03-24 200230](https://user-images.githubusercontent.com/85113567/159994056-2e71738b-6c3d-4d00-a4e7-03f229db7143.jpg)
- Figma design:
![Annotation 2022-03-24 200415](https://user-images.githubusercontent.com/85113567/159994087-7acd92b5-add2-46de-94a9-243e2301f286.jpg)